src/main/webapp/dataset-versions.xhtml

```diff
@@ -147,6 +147,25 @@
                 <h:outputText id="versionDate" value="#{versionTab.publicationDateAsString}" />
             </ui:fragment>
         </p:column><!-- end: date column -->
+        <!--  ToDo: change HeaderText to Bundled string, stop this fragment from being loaded for datasets during creation -->
```

Comment:
Does this todo still apply? it looks like you took care of both of those things in line 151.